nifi-nar-bundles/nifi-hazelcast-bundle/nifi-hazelcast-services/src/main/resources/org.apache.nifi.hazelcast.services.cacheclient.HazelcastMapCacheClient/additionalDetails.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Additional Details pages are not available on the UI.
These files should be in `resources/docs/...`